File: server/Models/Recipe.cs
namespace all_spice.Models;

public class Recipe 
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime MyProperty { get; set; }
    public string Title { get; set; }
    public string Instructions { get; set; }
    public string Img { get; set; }
    public string Category { get; set; }
    public string CreatorId { get; set; }

}

File: client/src/models/Recipe.js
export class Recipe {
    constructor(data){
        this.id = data.id
        this.createdAt = new Date(data.createdAt)
        this.updatedAt = new Date(data.updatedAt)
        this.title = data.title
        this.instructions = data.instructions
        this.img = data.img
        this.category = data.category
        this.creatorId = data.creatorId
    }
}